handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -61,6 +67,11 @@
 public class LoadbalanceHandler implements Handler {
   public static final String CONTEXT_KEY_SERVER_LIST = "x-context-server-list";
 
+  public static final String SERVICECOMB_SERVER_ENDPOINT = "scb-endpoint";
+
+  public static final boolean supportConfigEndpoint =
+      DynamicPropertyFactory.getInstance().getBooleanProperty("servicecomb.configEndpoint.enabled", false).get();
```

Comment:
Maybe we can use: servicecomb.loadbalance.userDefinedEndpoint.enabled

Reply:
done